### PR TITLE
Implemented anatomic incrementing of key version in Memcache(d)

### PR DESCRIPTION
**Pull request by SeBsZ**
to prevent race conditions

### CODE REVIEW

File: Cache_Memcache.php [hunk 1]
```diff
@@ -176,9 +176,7 @@ function hard_delete( $key, $group = '' ) {
 	 * @return boolean
 	 */
 	function flush( $group = '' ) {
-		$this->_get_key_version( $group );   // initialize $this->_key_version
-		$this->_key_version[$group]++;
-		$this->_set_key_version( $this->_key_version[$group], $group );
+        $this->_increment_key_version($group);
```
Comment (cssjoe) [inline Cache_Memcache.php:179]: As per WordPress code standards, please format using tabs to indent and spaces around parameters.
`		$this->_increment_key_version( $group );`

**Review by cssjoe — CHANGES_REQUESTED** [review]
Please format the new/updated lines of code to adhere to WordPress coding standards -- https://make.wordpress.org/core/handbook/best-practices/coding-standards/

Use tabs to indent rather than spaces.  Pad function parameters with one space.

Thank you.

**Comment by SeBsZ** [comment]
I fixed the tabs and spaces

**Review by maxicus — APPROVED** [review]

**Review by cssjoe — APPROVED** [review]

**Comment by cssjoe** [comment]
Thank you @SeBsZ

**Pull request merged**